### PR TITLE
l10n: fr.po: Review French translation

### PR DESCRIPTION
**Pull request by cmalard**

### CODE REVIEW

**Comment by submitgit** [comment]
@cmalard sent this commit (https://github.com/git/git/compare/1bc8feaa7cc752fe3b902ccf83ae9332e40921db...f77f14dee110af8ecd3790d712f9f65334c5cccf) as a patch to the mailing list with [_submitGit_](https://github.com/rtyley/submitgit) - here on [Gmane](http://mid.gmane.org/000001513f785287-0917c310-be63-4168-8d73-9bf4c53336fd-000000@eu-west-1.amazonses.com), [MARC](http://marc.info/?i=000001513f785287-0917c310-be63-4168-8d73-9bf4c53336fd-000000@eu-west-1.amazonses.com) [](000001513f785287-0917c310-be63-4168-8d73-9bf4c53336fd-000000@eu-west-1.amazonses.com)

**Comment by cmalard** [comment]
https://github.com/git/git/pull/185 was included

**Comment by nanalove28** [comment]
Nice

**Comment by kdurant** [comment]
@cmalard Can you help me to developers response a question?
I set:
```
	commitencoding = utf-8
	logoutputencoding = utf-8
```
**commit** can be displayed correctly in Chinese, but **tag** can't displayed properly in Chinese

**Comment by Bereiche54** [comment]
Meta Tags and Meta Keywords 


Meta Tags are thought to be the be all-end all of search engine rankings. This is not true. Meta Tags are a necessary, but not a sufficient condition for high rankings. Meta tags are designed to provide basic information to the search engine about the site. 

Meta tags appear at the top of the HTML document within the <head></head> section of the document. In the following example, nine different types of Meta Tags are identified: Content-Type, Copyright, Resource-type, ObjectType, Author, Rating, Distribution, Description, and Keywords. 

The two meta tags of primary interest to search engines are the “Description” and “Keywords” tags. These tags describe the contents and key words for the search engine. 

<HTML> 
<head> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"> 
<meta http-equiv="Copyright" content="2015- 2020 Bereiche Vergeichen Protocol ."> 
<meta name="Resource-type" content="Document"> 
<meta name="ObjectType" content="Journal"> 
<meta name="Author" content="Car Pictures .com Webmaster"> 
<meta name="Rating" content="General"> 
<meta name="Distribution" content="Global"> 
<meta name="Description" content="Gallery of car pictures, car videos, car sounds, car photos, car photographs, automobile pictures, automobile videos, automobile sounds, automobile photos, automobile photographs, auto pictures, auto videos, auto sounds, auto photos, auto photographs, pictures of cars."> 
<meta name="Keywords" content="Gallery of car pictures, car videos, car sounds, car photos, car photographs, automobile pictures, automobile videos, automobile sounds, automobile photos, automobile photographs, auto pictures, auto videos, auto sounds, auto photos, auto photographs, pictures of cars, Porsche pictures, Ferrari pictures, Lamborghini pictures."> 
<title> 
Car Pictures .com -- Gallery of pictures (photos or photographs), video, sounds, wallpaper and specifications of all cars, motorcycles and all vehicles. 
</title> 
</HEAD> 



Several excellent sources that detail the use of the Meta Tag are: 
The Web Developer’s Virtual Library (http://wdvl.com/Authoring/HTML/Head/Meta/ ) 
Search Engine Watch (http://searchenginewatch.internet.com/webmasters/meta.html)
Internet Marketing Chapter 13: Driving Traffic to Your Site: Understanding and Using Search Engines for Search and Site Optimization






Adding Keywords Using the META Tag 

Adding keywords simply requires editing the HTML code and adding the meta tags as in the following example The top of a web page source code on your site might look like: 

<HTML> 
<HEAD> 
<TITLE>ACME Corporation Devices Information</TITLE> 
</HEAD> 
<BODY> 
<H1>ACME Corporation Innovative Opening Devices Information</H1> 

Create a META tag with the following information: 

<META name="keywords" content="brochure"> 




TO ADD ADDITIONAL KEYWORDS 

<META name="keywords" content="brochure, widget, wholesale"> 

The top of your HTML document would then look like: 

<HTML> 
<HEAD> 
<TITLE>ACME Corporation Devices Information</TITLE> 
<META name="keywords" content="brochure, widget, wholesale"> 
</HEAD> 
<BODY> 
<H1>ACME Corporation Innovative Devices Information</H1> 


QUESTIONS AND ANSWERS ABOUT META KEYWORDS 

META keywords tag - should I use UPPERCASE, lowercase or a mixture, commas or spaces? 
Use lowercase, separated by commas, with a space after the comma



If I have several keyword phrases in my META keywords tag that all contain the same keyword, will I get penalized? For example: 

<META NAME="keywords" CONTENT="gold jewelry, diamond jewelry, silver jewelry, platinum jewelry"> This would represent four repetitions of the word 'jewelry'. 
I personally do not believe in spamming or putting multiple keywords throughout the page This is too easy to check for and the search engines are not stupid  If we can think of it they already have  Most engines only count to 2 for keywords so why waste the keyword utilization?Put in other keywords




Will one be penalized for using the same keywords on a lot of different pages that have the same content? If the pages have exactly the same content, why are they in your site? If they are all related to the same subject they can logically have the same keywords but the description tag would be different




What is the optimum number of keywords before the engine gets bored? 
It varies. Some are more easily bored than others I don't usually repeat keywords maybe twice for the important ones. It's better to show up adequately on all the engines than perfectly on one and badly on the rest




 Will the engines treat a phrase such as handmade gifts that is written as a keyword phrase the same as if it were written as two different words? Should I write them out both ways? 
 Use: "hand made handmade gift gifts" to cover all the possible variations searchers might enter




 

Tags:         Search Engine         Photos         Pictures         Photographs         Videos         Gallery         Search Engines         Html         Auto        


Keywords: Chinese 

    


Important Note: If you'd like to save a copy of the commentary on your computer, you can COPY and PASTE it into your word processor. Please, follow these steps to do that in Windows:
1. Select the text of the paper with the mouse and press Ctrl+C.
2. Open your word processor and press Ctrl+V.

Company's Liability 

123HelpMe.com (the "Web Site") is produced by the "Company". The contents of this Web Site, such as text, graphics, images, audio, video and all other material ("Material"

**Comment by dscho** [comment]
@cmalard is this PR still relevant? If so, please rebase your PR, and then open a PR either in the indicated fork or in https://github.com/git-l10n/git-po.

**Comment by cmalard** [comment]
Wow. No idea, this PR is so old x'D

@dscho : what's the indicated fork? I can simply close this PR and make a new one in git-po if it's still relevant and simpler.

**Comment by dscho** [comment]
> what's the indicated fork?

Whoops, I meant https://github.com/git-l10n/git-po by "the indicated fork".

> I can simply close this PR and make a new one in git-po if it's still relevant and simpler.

Please rebase to v2.22.0-rc3 first, after that, I am sure that a PR would be welcome there. (Sadly, the Git project does not accept PRs, but Git's translation project does.)

**Comment by cmalard** [comment]
@jnavila : not sure how I should deal with this and https://github.com/jnavila/git ? :-)

**Comment by jnavila** [comment]
The meat of the PR may still be relevant, but the patch won't apply. Can you port your changes to the latest master on my repo and open a new PR? Don't forget to include a signed-off line.

File: po/fr.po [hunk 1]
```diff
@@ -13145,12 +13145,12 @@ msgstr "impossible de faire un stat du modèle '%s'"
 #: builtin/init-db.c:66
 #, c-format
 msgid "cannot opendir '%s'"
-msgstr "impossible d'ouvrir (opendir) '%s'"
```
Comment (jnavila) [inline po/fr.po]: Pour ceux-ci, je suis partagé... le texte d'origine cherche vraiment à citer le nom de la fonction qui a échoué. Doit-on éliminer cette référence ?

Comment (cmalard) [reply po/fr.po]: Je me suis aussi posé la question, j'ai pris parti sur l'autre ref d'opendir qui était traduite :
https://github.com/git/git/blob/0c0ea992e8a9155f6b44ad7eb1a92b9b19042c38/po/fr.po#L22200
```po
#: git-send-email.perl:637
#, perl-format
msgid "Failed to opendir %s: %s"
msgstr "Échec à l'ouverture du répertoire %s : %s"
```
Je n'ai pas l'impression qu'il y ait une plus-value à garder le nom de la fonction explicite, si ?

Comment (jnavila) [reply po/fr.po]: Je ne sais pas, c'est en perl.

**Comment by jnavila** [comment]
I just pushed a PR of your patch to the main maintainer's repo: https://github.com/git-l10n/git-po/pull/375

**Comment by jnavila** [comment]
@cmalard please close this PR.

**Pull request closed without merging**